sites/cheerpx/src/pages/index.astro

```diff
@@ -1,60 +1,183 @@
-				label="Try the demo"
-			/>
+		<div class="flex justify-center items-center">
+			<LinkButton type="primary-cheerpx" href="#youtube" label="Learn more" />
 		</div>
 	</section>
-	<section class="flex justify-center">
-		<iframe
-			title="CheerpX: a WebAssembly-based x86 virtual machine in the browser, Yuri Iozzelli"
-			src="https://youtube.com/embed/VqrbVycTXmw?start=151"
-			class="w-full max-w-[720px] aspect-video rounded-xl shadow-lg border-2 border-stone-700"
-			frameborder="0"
-			allowfullscreen="allowfullscreen"></iframe>
-	</section>
-</ProductHome>
 
-<style>
-	@keyframes flyFadeScaleIn {
-		0% {
-			opacity: 0;
-			transform: translateY(70px) scale(0.6);
-		}
-		100% {
-			opacity: 1;
-			transform: translateY(0) scale(1);
-		}
-	}
+	<!-- TODO: video -->
+	<!-- TODO: client company logos -->
+	<!-- TODO: how it works (user story) -->
+	<!-- TODO: testimonials -->
+	<!-- TODO: FAQ -->
+	<section id="youtube" class="bg-black py-32 px-8">
+		<p class="text-center text-bold text-5xl py-2">
+			x86 virtualization in the browser
+		</p>
+		<div
+			class="flex justify-center px-4 sm:p-8 md:px-16 lg:px-32 xl:px-56 2xl:px-60"
+		>
+			<iframe
+				title="CheerpX: a WebAssembly-based x86 virtual machine in the browser, Yuri Iozzelli"
+				src="https://youtube.com/embed/VqrbVycTXmw?start=151"
+				class="w-full max-w-[720px] aspect-video rounded-xl shadow-lg border-2 border-stone-700"
+				frameborder="0"
+				allowfullscreen="allowfullscreen"
+			>
+			</iframe>
+		</div>
+	</section>
+	<section
+		class="bg-gradient-to-b from-black to-[#00254d] pb-48 text-center pb-28 md:px-8 md:pt-16 lg:px-32 lg:pt-32 xl:pt-42 xl:px-28 2xl:px-60 2xl:pt-60"
+	>
+		<p class="text-center text-bold text-5xl py-20">
+			Native code virtualization as a JavaScript library
+		</p>
 
-	.fly-in-b {
-		animation: flyFadeScaleIn 300ms cubic-bezier(0.165, 0.84, 0.44, 1) forwards;
-	}
-</style>
+		<div class="px-16 grid sm:grid-cols-1 md:grid-cols-3 space-x-4">
+			<div class="col-span-2 text-left font-bold text-2xl py-4">
+				<p>
+					<span class="text-white">Do you have a commercial use case?</span>
+				</p>
+				<ul class="list-inside list-disc font-normal text-wrap text-lg py-4">
+					<li>Technical support with tailored SLAs.</li>
+					<li>Feature prioritisation and sponsored development available.</li>
+					<li>
+						Access to our development team for compilation log reviews, video
+						calls and consulting.
+					</li>
+				</ul>
+				<LinkButton
+					type="primary-cheerpx"
+					href="https://leaningtech.com/contact-us/?subject=Cheerp%20Enterprise%20Licence#mailus"
+					label="Contact us"
+				/>
+			</div>
+			<div class="col-span-1 border-solid border-l text-left py-4">
+				<p class="text-left text-2xl font-bold text-stone-300 pl-8">
+					<span class="text-white">Ready to build?</span> Get started today by browsing
+					our
+					<a
+						href="/docs"
+						class="text-yellow-600 hover:text-white inline-flex items-center"
```

Comment:
`class="text-yellow-600` You may have forgotten to change the font colour here, it's still in orange, you can pick a font colour you feel appropriate here: https://tailwindcss.com/docs/text-color